README.md

```diff
@@ -20,9 +20,7 @@ bundle install
 touch .env
 
 # Clone production data
-rake db:create db:migrate
-rake cascade:sync
-rake faculty:sync
+cap production db:clone
```

Comment:
That said, "because we've always done it that way" is not always a great justification. I'd still like to know why we did it that way in the Inside app.